OpenRA.Mods.Common/Activities/Air/FlyAttack.cs

```diff
@@ -36,12 +36,12 @@ public FlyAttack(Actor self, Target target)
 		public override Activity Tick(Actor self)
 		{
 			if (!target.IsValidFor(self))
-				return NextActivity;
+				return new ReturnToBase(self);
```

Comment:
This is to make the aircraft try to return to the resupply base even if its already reserved after destroying a target. It should be better to preserve the NextActivity? Maybe but it on a Activity Sequence that the ReturnToBase is the last?
I know for sure that when there is no more commands, NextActivity is null. Maybe only return to base is this case?